src/engine/shared/config_variables.h

```diff
@@ -65,6 +65,7 @@ MACRO_CONFIG_INT(InfSoldierBombs, inf_soldier_bombs, 3, 1, 16, CFGFLAG_SERVER, "
 MACRO_CONFIG_INT(InfMercBombs, inf_merc_bombs, 15, 1, 100, CFGFLAG_SERVER, "Number of poison grenades for the mercenary")
 MACRO_CONFIG_INT(InfMercLove, inf_merc_love, 1, 0, 1, CFGFLAG_SERVER, "Enables love bombs for the mercenary (hammer)")
 MACRO_CONFIG_INT(InfBarrierLifeSpan, inf_barrier_lifespan, 30, 0, 100, CFGFLAG_SERVER, "Barrier lifespan")
+MACRO_CONFIG_INT(InfVoodooAliveTime, inf_voodoo_alive_time, 30, 0, 5, CFGFLAG_SERVER, "How long a voodoo keeps staying alive after being killed")
```

Comment:
sorry, i don't understand this. My understanding was that the first int value is the default, second is "minimum" and third is "maximum". Is it ok that default voodoo alive time is 30 while max is 5?